lib/streamlit/DeltaGenerator.py

```diff
@@ -72,30 +72,20 @@ def _wraps_with_cleaned_sig(wrapped, num_args_to_remove):
     num_args_to_remove). This is useful since function signatures are visible
     in our user-facing docs, and many methods in DeltaGenerator have arguments
     that users have no access to.
+
+    Note that "self" is ignored by default. So to remove both "self" and the
+    next argument you'd pass num_args_to_remove=1.
```

Comment:
This behavior is new. Not sure why `functools.partial` is removing `self` now, but it definitely is.